components/assets/README.md

```diff
@@ -0,0 +1,20 @@
+# Assets
```

Comment:
Not sure if we want to call this folder assets. We already have an asset folder that actually contains assets, so this might cause some confusion. Maybe call it loaders or importers?